app/styles/ui/history/_history.scss

```diff
@@ -80,16 +81,40 @@
     margin-top: var(--spacing);
   }
 
-  .branches-list-item .description {
-    margin-right: 0;
+  .branches-list-item {
+    .description {
+      margin-right: 0;
+    }
+
+    .name {
+      max-width: 60%;
```

Comment:
This isn't immediately clear to me. What do we gain by capping the max width as opposed to disabling shrinking of the description? Is this some workaround to get truncation to work in the flexbox?

Reply:
Whoops, this was left over from some stuff I was doing trying to get the commit counters to look good by capping how wide the branch name gets. 😅Turns out it didn't make any real difference in the end. Removed in b496c91

Here's what it looks like without that style:

![screen shot 2018-04-03 at 9 29 13 am](https://user-images.githubusercontent.com/1174461/38262407-f3d66e52-3721-11e8-9176-fff58d66578d.png)